mn/widget.js

```diff
@@ -0,0 +1,271 @@
+    'emoji-picker.travel': 'Газар',
+    'emoji-picker.objects': 'Обьект',
+    'emoji-picker.symbols': 'Тэмдэгт',
+    'emoji-picker.flags': 'Далбаа',
+
+    /**
+     * GIF-picker labels
+     */
+    'gif-picker.search': 'GIF хайх',
+
+    /**
+     * Typing
+     */
+    'typing.a': '1 хүн бичиж байна...',
+
+    'typing.b': '$N хүн бичиж байна...',
+    // example: "5 people are typing..."
+
+    'typing.c': '$A бичиж байна...',
+    // example: "Alex is typing..."
+
+    'typing.d': '$A, $B нар бичиж байна...',
+    // example: "Alex and Ben are typing..."
+
+    'typing.e': '$A, $B, $C нар бичиж байна...',
+    // example: "Alex, Ben, and Cindy are typing..."
+
+    'typing.f': '$A болон өөр 1 хүн бичиж байна...',
+    // example: "Alex and 1 other are typing..."
+
+    'typing.g': '$A болон өөр $N хүн бичиж байна...',
+    // example: "Alex and 3 others are typing..."
+
+    'typing.h': '$A, $B, өөр 1 хүн бичиж байна...',
+    // example: "Alex, Ben, and 1 other are typing..."
+
+    'typing.i': '$A, $B, өөр $N хүн бичиж байна...',
+    // example: "Alex, Ben, and 3 others are typing..."
+
+    'typing.j': '$A, $B, $C, өөр 1 хүн бичиж байна...',
+    // example: "Alex, Ben, Cindy, and 1 other are typing..."
+
+    'typing.k': '$A, $B, $C, өөр $N хүн бичиж байна...',
+    // example: // example: "Alex, Ben, Cindy, and 3 others are typing..."
+
+    /**
+     * Social login dialog
+     */
+    'social-modal.title': 'Нэвтрэх',
+    'social-modal.header': 'Now4real ашигладаг сайтад чатлаж эхлэх',
+
+    // buttons:
+    'social-modal.google': 'Google аккаунтаар нэвтрэх',
+    'social-modal.facebook': 'Facebook аккаунтаар нэвтрэх',
+    'social-modal.twitter': 'Twitter аккаунтаар нэвтрэх',
+    'social-modal.linkedin': 'LinkedIn аккаунтаар нэвтрэх',
+    'social-modal.email': 'Мэйл хаягаар нэвтрэх',
+
+    'social-modal.footer': 'Та Now4real-ийн <a1>Үйлчилгээний нөхцөл</a1>, <a2>Нууцлалын бодлогыг</a2> зөвшөөрч байгаа бол "Нэвтрэх" товчийг дарна уу',
+    // keep <a1> and </a1> unchanged and surrounding the translation of "Terms of Service"
+    // keep <a2> and </a2> unchanged and surrounding the translation of "Privacy Policy"
+
+    /**
+     * JWT login dialog
+     */
+    'jwt-modal.title': 'Зөвшөөрөл',
+
+    'jwt-modal.header': 'Чаатыг ашиглахын тулд Та Now4real-ийн <a1>Үйлчилгээний нөхцөл</a1> ба <a2>БНууцлалын бодлогыг</a2> зөвшөөрөх ёстой',
+    // keep <a1> and </a1> unchanged and surrounding the translation of "Terms of Service"
+    // keep <a2> and </a2> unchanged and surrounding the translation of "Privacy Policy"
+
+    'jwt-modal.button': 'Зөвшөөрөх',
+
+    /**
+     * Mute dialog
+     */
+    'mute-modal.title': 'Хэрэглэгчийг хаах',
+
+    'mute-modal.header': 'Хэрэглэгчийн эрх $U $E хүртэл хаагдсан',
+    // example: "User Ben has been muted until Wed, 14 Jun 2022 07:12:00 GMT"
+
+    'mute-modal.headerForever': 'Хэрэглэгч $U-г хаасан',
+
+    'mute-modal.body.set': 'Хэрэглэгч $U-г хаах хугацаа:',
```

Comment:
The English verb is "to mute", with the meaning of "block the user for posting new messages"